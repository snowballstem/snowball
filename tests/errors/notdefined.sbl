externals (stem)
groupings (g)
routines (r)
define stem as (g r)
