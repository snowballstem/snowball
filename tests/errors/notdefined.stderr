errors/notdefined.sbl:4: routine 'r' declared but not defined
errors/notdefined.sbl:4: grouping 'g' declared but not defined
